### ISSUE TITLE
Please don't propose to use the flagship instance by default

### ISSUE THREAD

**Issue by stemy2** [mastodon/mastodon-android#568]
Hello,

Prompting users to join mastodon.social by default instead of let users picking a server could be highly harmful to mastodon and the whole fediverse. If a single instance grows too much, it reduces the decentralisation and make the said instance vulnerable to a takeover.

So i think that for the good of the fediverse, you should remove a such feature.

**Comment by tali-vitali** [comment]
yes please. alternative can be randomizing server suggestion, not only flagship one. mb from list of general common stable servers, something like this

**Comment by antimundo** [comment]
I think it's good that the default options come from a curated list, rather than showing literally all available servers in the fediverse, as a new user could end up on a poorly managed server and have a bad experience.

However, I think that instead of just sending everyone to mastodon.social, there could be a curated list with some well-moderated servers.

**Comment by FediVideos** [comment]
Mastodon.social is now approaching 30% of all active users, and new people are assuming they have to sign up there because of how the official apps are promoting mastodon.social. It's totally possible it could get to over 50% of the Fediverse, at which point the Fedi would no longer be decentralised.

Could we please switch the official Mastodon app to recommending a featured server from a trusted pool of servers instead of mastodon.social? As I recommended over on the iOS app, mas.to has very similar policies and a reliable track record going back years, but mastodon.social is twenty-four times bigger. Surely it's time to promote other reliable servers now?

**Comment by JonatanGalAlv** [comment]
As all the people said before, making dot social as default not only makes the whole decentralized social network just in theory, it also makes it less resistant to any attack (censorship, bots... You name it)

**Comment by fatkudu** [comment]
Another vote for avoiding centralization.  A rotating lottery from a set of good options seems fairer.

**Comment by mitradranirban** [comment]
Strongly support
